<launch>
	<param name="/use_sim_time" value="true" />
	<include file="$(find navigation_stage)/move_base_config/move_base.xml" />
	<node name="map_server" pkg="map_server" type="map_server"
		args="$(find navigation_stage)/stage_config/maps/willow-full.pgm 0.1"
		respawn="false" />
	<node pkg="stage" type="stageros" name="stageros"
		args="$(find navigation_stage)/stage_config/worlds/willow-pr2.world"
		respawn="false">
		<param name="base_watchdog_timeout" value="0.2" />
	</node>
	<node name="fake_localization" pkg="fake_localization" type="fake_localization"
		respawn="false" />

	<node pkg="delayed_teleop" type="simulated_delayed_comunication.py"
		name="delayed_msgs">

		<rosparam command="load"
			file="$(find delayed_teleop)/launch/demo_config.yaml" />
	</node>
	<!-- <rosparam command="load" ns="delayed_msgs" file="$(find delayed_teleop)/launch/demo_config.yaml" 
		/> -->

	<node name="rviz" pkg="rviz" type="rviz"
		args="-d $(find delayed_teleop)/config/rviz-config.vcg" />

	<node pkg="joy" type="joy_node" name="joy"></node>
	<node pkg="delayed_teleop" type="teleop_joy" name="teleop_joy">
		<param name="axis_linear" value="1"></param>
		<param name="axis_angular" value="0"></param>
		<param name="scale_angular" value="2.5"></param>
		<param name="scale_linear" value="2.5"></param>
		<param name="robot_base_link" value="tf_delayed_base_link" />
	</node>

</launch>